internal/setup/setup.go

```diff
@@ -344,6 +344,13 @@ type yamlRelease struct {
 	V1PubKeys map[string]yamlPubKey `yaml:"v1-public-keys"`
 }
 
+const (
+	// An archive's priority value cannot be larger than MaxArchivePriority.
+	MaxArchivePriority = 1000000
```

Comment:
Let's make this `1000` until we have the discussion with Gustavo because `1000000` is definitely too big.

Reply:
Okies. Changed in eef309d.